tests/containers/nccl-tests/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG CUDA_VERSION_MINOR=12.8.1
+ARG BASE_IMAGE=nvidia/cuda:${CUDA_VERSION_MINOR}-devel-ubuntu24.04
+FROM ${BASE_IMAGE} as base
+
+ARG CUDA_VERSION_MAJOR=12.8
+ARG TARGET_NCCL_VERSION=2.25.1-1
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get -qq update && \
+    apt-get -qq install -y \
+    --allow-change-held-packages \
+    --no-install-recommends \
+    --allow-downgrades \
+    build-essential libtool autoconf automake autotools-dev unzip \
+    ca-certificates \
+    wget curl openssh-server vim environment-modules \
+    iputils-ping net-tools \
+    libnuma1 libsubunit0 libpci-dev \
+    libpmix-dev \
+    datacenter-gpu-manager \
+    g++ libopenmpi-dev openmpi-bin \
+    libnccl2=$TARGET_NCCL_VERSION+cuda${CUDA_VERSION_MAJOR} \
+    libnccl-dev=${TARGET_NCCL_VERSION}+cuda${CUDA_VERSION_MAJOR} \
+    git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+    ibverbs-utils libibverbs-dev libibumad3 libibumad-dev librdmacm-dev rdmacm-utils infiniband-diags ibverbs-utils \
```

Comment:
Do we have plans to replace the ibtools image with this nccl-tests image? I see a lot of package overlap between them

Reply:
I do plan to remove that tests and the image. Now that we use mpirun and the ib tests are enough, IMO.